### DISCUSSION TITLE
Integration with tailwindcss

### DISCUSSION THREAD

**Discussion by ricardopacheco** [shakacode/shakapacker#361] [General]
## Expected behavior:

From the official documentation (shakapacker and tailwind itself), I was able to run tailwindcss completely functionally.

## Actual behavior:

It works fully when used with other methods such as esbuild and using nextjs for example (which I ran following the official documentation of tailwindcss itself). Tailwind partially works. Other people have same problem: https://stackoverflow.com/questions/73311322/tailwind-3-not-loading-completely-in-rails-shakapacker-haml-webpackerv6-w

## Small, reproducible repo:

Follow the official documentation (shakapacker and tailwindcss) in versions above.

## Setup environment:
- Ruby version:  3.2.2
- Rails version: 7.0.8
- Shakapacker version:  7.0.3
- Tailwindcss: 3.3.3

**Comment by ahangarha** [comment]
We already have applications using Shakapacker and Tailwind CSS. So, there is no integration issue.

Please share a repo so we can help you fix your issue.

**Comment by ricardopacheco** [comment]
Sorry guys, it was a docker issue. I hadn't restarted the container after the adjustments and that's why it didn't work. Both docs are ok and following them works.

Sorry for the inconvenience.

**Reply by ahangarha** [reply]
Thanks for sharing the update.

We would be happy to share if you can write an article about such a combination.

**Reply by ricardopacheco** [reply]
@ahangarha Excelent idea. I must create a personal blog and I already have my first practical article. Thank you very much!!
